src/classes/Recipe.js

```diff
@@ -1,39 +1,44 @@
 class Recipe {
-    constructor(recipeDetails = {}) {
-        this.id = recipeDetails.id || this.generateRandomId();
-        this.name = recipeDetails.name;
-        this.imageURL = recipeDetails.imageURL;
-        this.portions = recipeDetails.portions || [];
-        this.instructions = recipeDetails.instructions || [];
-        this.tags = recipeDetails.tags || [];
+  constructor(recipeDetails = {}) {
+    this.id = recipeDetails.id || this.generateRandomId();
+    this.name = recipeDetails.name;
+    this.imageURL = recipeDetails.imageURL;
+    this.portions = recipeDetails.portions || [];
+    this.instructions = recipeDetails.instructions || [];
+    this.tags = recipeDetails.tags || [];
+  }
+  generateRandomId() {
+    return Math.floor(Math.random() * 1000);
+  }
+
+  getPortionNames() {
+    return this.portions.map((portion) => {
+      return portion.name;
+    });
+  }
 
-    }
-    generateRandomId() {
-        return Math.floor(Math.random() * 1000);
-    }
+  getPortionCosts() {
+    const ingredientsCost = this.portions.map((portion) => {
+      return (portion.amount * portion.cost).toFixed(2) / 100;
+    });
+    return ingredientsCost;
+  }
+
+  getInstructions() {
+    return this.instructions;
+  }
 
-    getPortionNames() {
-        return this.portions.map((portion) => {
-            return portion.name;
-        });
-    }
-    getPortionCosts() {
-        const ingredientsCost = this.portions.map((portion) => {
-            return (portion.amount * portion.cost).toFixed(2) / 100;
```

Comment:
I wonder if toFixed() is working here or not. DOM has a many more than 2 decimals